quickstart/cpp/windows/text-to-speech/README.md

```diff
@@ -1,11 +1,14 @@
 # Quickstart: Synthesize speech in C++ for Windows
 
 This sample demonstrates how to synthesize speech with C++ using the Speech SDK for Windows.
-See the [accompanying article](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-text-to-speech-cpp-windows) on the SDK documentation page which describes how to build this sample from scratch in Visual Studio 2017.
+See the [accompanying article](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/get-started-text-to-speech?tabs=terminal&pivots=programming-language-cpp) on the SDK documentation page which describes how to build this sample from scratch in Visual Studio 2017.
 
 ## Prerequisites
 
-* A subscription key for the Speech service. See [Try the speech service for free](https://docs.microsoft.com/azure/cognitive-services/speech-service/get-started).
+* * A subscription key and region for the Speech service. 
+  * Azure subscription - [Create one for free](https://azure.microsoft.com/en-us/free/cognitive-services/)
+  * [Create a Speech resource](https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesSpeechServices) in the Azure portal.
+  * Get the resource key and region. After your Speech resource is deployed, select Go to resource to view and manage keys. or  more information about Cognitive Services resources, see [Get the keys for your resource](https://docs.microsoft.com/en-us/azure/cognitive-services/cognitive-services-apis-create-account?tabs=multiservice%2Cwindows#get-the-keys-for-your-resource)
```

Comment:
"Go to resource" is bold on the docs.microsoft.com view for this and I think it's important to visually distinguish it.

Could you please update all instances of this with enclosing double asterisks? That'll make it bold, e.g.:
\*\*this text\*\*
becomes **this text**